internal/appmetrics/counters.go

```diff
@@ -31,4 +31,30 @@ var (
 		Name: "devices_api_blackbook_request_success_ops_total",
 		Help: "Total successful Blackbook used",
 	})
+	// Chat GPT Metrics
+	OpenAITotalCallsOps = promauto.NewCounter(prometheus.CounterOpts{
+		Name:      "devices_api_openai_chatgpt_ops_total",
+		Help:      "Total number of calls to Open AI ChatGPT",
+		Namespace: "devices_api",
+		Subsystem: "error_codes",
+	})
+	OpenAITotalFailedCallsOps = promauto.NewCounter(prometheus.CounterOpts{
+		Name:      "devices_api_openai_chatgpt_ops_failed_calls_total",
```

Comment:
This is super picky but `ops` doesn't seem to be doing much here. `devices_api_error_codes_openai_failed_requests_total`?

I mostly borrowed naming conventions from [here](https://prometheus.io/docs/practices/naming/) and [here](https://prometheus.io/docs/practices/instrumentation/#failures).